WordPress/src/main/java/org/wordpress/android/ui/media/MediaAddFragment.java

```diff
@@ -102,6 +118,9 @@ public void onReceive(Context context, Intent intent) {
                     ToastUtils.showToast(context, errorMessage, ToastUtils.Duration.SHORT);
                 }
                 mCallback.onMediaAdded(mediaId);
+            } else {
```

Comment:
A network check has already been added in `startMediaUploadService()`, do we still want another check in here?